src/backend/distributed/utils/maintenanced.c

```diff
@@ -271,66 +339,97 @@ WarnMaintenanceDaemonNotStarted(void)
 
 
 /*
- * CitusMaintenanceDaemonMain is the maintenance daemon's main routine, it'll
- * be started by the background worker infrastructure.  If it errors out,
- * it'll be restarted after a few seconds.
+ * ConnectToDatabase connects to the database for the given databaseOid.
+ * if databaseOid is 0, connects to MainDb and then creates a hash entry.
+ * If a hash entry cannot be created for MainDb it exists the process requesting a restart.
+ * However for regular databases, it exits without requesting a restart since another
+ * subsequent backend should be starting the Maintenance Deamon.
+ * If a hash entry found has a valid workerPid, it exits
+ * without requesting a restart since there is already a deamon running.
```

Comment:
rewording + typo fixes

```suggestion
 * ConnectToDatabase connects to the database for the given databaseOid.
 * if databaseOid is 0, connects to MainDb and then creates a hash entry.
 * If a hash entry cannot be created for MainDb it exists the process requesting a restart.
 * However for regular databases, it exits without requesting a restart since another
 * subsequent backend should be starting the Maintenance Daemon.
 * If the found hash entry has a valid workerPid, it exits
 * without requesting a restart since there is already a daemon running.
```